docs/products/kafka/kafka-connect/howto/splunk-sink.rst

```diff
@@ -53,7 +53,7 @@ Create a file (we'll refer to this one as ``splunk_sink.json``) to hold the conn
         "topics": "TOPIC_LIST",
         "splunk.hec.raw" : false,
         "splunk.hec.ack.enabled" : false,
-        "splunk.hec.ssl.validate.certs": "false",
+        "splunk.hec.ssl.validate.certs": "true",
```

Comment:
Definitely needs to be false, we have some self-signed certs in use.

Reply:
mmmm, I see the note later, so let's keep this but I think it may trip people ...